core/extension/gdextension.cpp

```diff
@@ -41,6 +41,11 @@ extern GDExtensionInterfaceFunctionPtr gdextension_get_proc_address(const char *
 
 typedef GDExtensionBool (*GDExtensionLegacyInitializationFunction)(void *p_interface, GDExtensionClassLibraryPtr p_library, GDExtensionInitialization *r_initialization);
 
+#ifdef WINDOWS_ENABLED
```

Comment:
Should like be this to match where it's used:
```suggestion
#if defined(TOOLS_ENABLED) && defined(WINDOWS_ENABLED)
```